src/csvStringToObject.md

```diff
@@ -24,3 +26,11 @@ const CSV_to_JSON = (data, delimiter = ',') => {
 };
 ```
 
+### Python
+```py
+def csv_to_json(data, delimiter=','):
+    csv_data = io.StringIO(data)
```

Comment:
Do not use any external library, because if someone copies this function it might not work for them because of dependency

Reply:
See if you can do it without using any lib so the function can be used easily